ObjectMapper/Core/ToJSON.swift

```diff
@@ -121,7 +121,20 @@ internal final class ToJSON {
             objectArray(field, key: key, dictionary: &dictionary)
         }
     }
-    
+	
+	
+	class func objectSet<N: Mappable>(field: Set<N>, key: String, inout dictionary: [String : AnyObject]) {
```

Comment:
These methods should constrain the generic parameter as `<N: Mappable where N: Hashable>`.